src/components/timers/XY.tsx

```diff
@@ -1,3 +1,123 @@
-const XY = () => null;
+import { useState, useEffect } from 'react';
+import { userInputCleanup } from '../../utils/helpers';
+import HomeBtnsWithBack from '../generic/HomeBtnsWithBack';
+import NumberpadInput from '../generic/NumberpadInput';
+import TimerDisplay from '../generic/TimerDisplay';
 
+const XY = () => {
+    const [seconds, setSeconds] = useState(0);
+    const [isRunning, setIsRunning] = useState(false);
+    const [isInputSet, setIsInputSet] = useState(false);
+    const [isDone, setIsDone] = useState(false);
+    const [roundsRemaining, setRoundsRemaining] = useState(0);
+    const [roundDurationInput, setRoundDurationInput] = useState('0000');
+    const [roundAmountInput, setRoundAmountInput] = useState('00');
```

Comment:
Why is roundAmountInput a string instead of a number? You have control over the buttons (in other words, you know exactly what inputs are possible - they'll never be bad user input).

You could consider converting the innerText of the button to a number before passing it to setRoundAmountInput, or even better - you can add a wrapper around handleInputBtnClick in your NumberPadInput component, that passes in the value rather than the text to the handler.